install/bootstrap_livecd_usb.md

```diff
@@ -517,9 +517,9 @@ Now that the configuration is generated, the LiveCD must be populated with the g
 
    ```bash
    linux# echo "
-   CSM_PATH=/var/www/ephemeral/${CSM_RELEASE}
    CSM_RELEASE=${CSM_RELEASE}
-   PITDATA=/var/www/ephemeral
+   PITDATA=$(lsblk -o MOUNTPOINT -nr /dev/disk/by-label/PITDATA)
```

Comment:
That lsblk command will work and give us /var/www/ephemeral even though we aren't yet booted from the PIT, and PITDATA is still mounted over /mnt/pitdata? Since this is just being written into the environment file of the PIT node, we want it to have the correct value for the booted PIT, not its current value.

Reply:
Nice catch, thank you.

I've corrected this in my recent push.